blocks/nav/__list/nav__list.css

```diff
@@ -0,0 +1,18 @@
+.nav__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  transition: all .3s linear;
+  height: 0; 
+  overflow: hidden;
+}
+
+@media screen and (min-width:1100px) {
+  .nav__list {
```

Comment:
On the dimensions 1100px - 1248px this looks condensed on width. Check it and correct, please.